web/landing_page.html

```diff
@@ -30,6 +30,14 @@ <h1>{{.Name}}</h1>
       </div>
       {{ end }}
       {{ .ExtraHTML }}
+      <div id="pprof">
+      Go pprof debugging profile data:
+      <ul>
+        <li><a href="debug/pprof/heap">heap usage (memory)</a>
+        <li><a href="debug/pprof/profile">CPU usage (takes some time to gather)</a>
+      </ul>
+      Try posting these to <a href="https://pprof.me" target="_blank">pprof.me</a>
```

Comment:
```suggestion
       "To share with an expert, you can upload to <a href="https://pprof.me" target="_blank">pprof.me</a>
```

Reply:
Hmm, I don't like the `expert` term. I made a minor tweak to the suggestion.